workinggroups/networking.md

```diff
@@ -0,0 +1,53 @@
+# CNCF Networking WG Proposal
+
+## TOC Sponsor
+
+Ken Owens
+
+## Objective
+
+Explore cloud native networking technology and concepts around the container networking interface (CNI).
```

Comment:
"Explore" seems like an activity rather than an objective. Is there an objective at this point?